unlocks.csv

```diff
@@ -458,29 +458,29 @@ Goal|Goal_CuteFish|0,0,BaseWaterPark|Glass|Builder|Battery
 Goal|UnknownEgg|0,0,
 Goal|OnConstructExosuit|0,0,Exosuit|Constructor|TitaniumIngot|PlasteelIngot|Aerogel|EnameledGlass|Glass
 Goal|OnConstructSeamoth|0,0,Seamoth|Constructor|TitaniumIngot|PowerCell|Battery|Silicone|Glass|Lubricant
-Databox|Compass,1,
-Databox|BaseUpgradeConsole,1,
-Databox|PlasteelTank,1,LaserCutter
-Databox|CyclopsSonarModule,1,LaserCutter
-Databox|ReinforcedDiveSuit,1,
-Databox|BaseWaterPark,1,
-Databox|CyclopsDecoy,1,
-Databox|CyclopsDecoyModule,1,LaserCutter
-Databox|CyclopsFireSuppressionModule,1,LaserCutter
-Databox|RepulsionCannon,1,
-Databox|SwimChargeFins,1,
-Databox|CyclopsThermalReactorModule,1,LaserCutter
-Databox|StillSuit,1,
-Databox|CyclopsHullModule1,1,LaserCutter
-Databox|HighCapacityTank,1,
-Databox|CyclopsShieldModule,1,
-Databox|CyclopsSeamothRepairModule,1,LaserCutter
-Databox|UltraGlideFins,1,
-Rocket|0,0,Constructor|TitaniumIngot|ComputerChip|CopperWire
+Databox|Compass,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|BaseUpgradeConsole,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|PlasteelTank,1,BaseUpgradeConsole|CyclopsShieldModule|LaserCutter
+Databox|CyclopsSonarModule,1,BaseUpgradeConsole|CyclopsShieldModule|LaserCutter
+Databox|ReinforcedDiveSuit,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|BaseWaterPark,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|CyclopsDecoy,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|CyclopsDecoyModule,1,BaseUpgradeConsole|CyclopsShieldModule|LaserCutter
+Databox|CyclopsFireSuppressionModule,1,BaseUpgradeConsole|CyclopsShieldModule|LaserCutter
+Databox|RepulsionCannon,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|SwimChargeFins,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|CyclopsThermalReactorModule,1,BaseUpgradeConsole|CyclopsShieldModule|LaserCutter
+Databox|StillSuit,1,BaseUpgradeConsole|CyclopsShieldModule|RocketBase|RocketBaseLadder|RocketStage1|RocketStage2|RocketStage3|CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|HatchingEnzymes
+Databox|CyclopsHullModule1,1,BaseUpgradeConsole|CyclopsShieldModule|LaserCutter
+Databox|HighCapacityTank,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|CyclopsShieldModule,1,BaseUpgradeConsole|CyclopsShieldModule
+Databox|CyclopsSeamothRepairModule,1,BaseUpgradeConsol|CyclopsShieldModulee|LaserCutter
+Databox|UltraGlideFins,1,BaseUpgradeConsole|CyclopsShieldModule
+Rocket|0,1,Constructor|TitaniumIngot|ComputerChip|CopperWire|BaseUpgradeConsole|CyclopsShieldModule|RocketBase|RocketBaseLadder|RocketStage1|RocketStage2|RocketStage3|CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|HatchingEnzymes
```

Comment:
Since this never actually manifests in the game, and setting to non-standard did not prevent it getting picked up, just added important checks as softlock protection.